src/content/en/2021/capabilities.md

```diff
@@ -1,20 +1,234 @@
+
+1. `none` does not capture the link at all (the default)
+2. `new-client` opens a new window for the PWA
+3. `existing-client-navigate`  navigates an existing client to the new URL or opens a new window if no client exists
+
+Declarative Link Capturing also is an extension of the Web Application Manifest. To use it, you need to add the `capture_links` property to your manifest. This property takes a string or an array of strings matching the three modes from above. If you use an array, the browser will fall back to the next entry if it doesn’t support a particular mode.
+
+```json
+{
+  "capture_links": [
+    "existing-client-navigate",
+    "new-client",
+    "none"
+  ]
+}
+```
+
+This capability is at an early stage as well. It is only supported on Chrome OS. Currently, 36 desktop sites and 11 mobile sites use this capability, for example, [Periodex](https://periodex.co/), a PWA showing the periodic table of elements. This app uses the `capture_links` configuration as shown in the listing above: If supported, the browser should reuse the existing window, otherwise, open a new one, and if that’s not supported, it should behave as normal.
+
+### Hardware APIs
+The next set of capabilities focuses on hardware-related APIs. In Chromium-based browsers, there are many APIs to access hardware interfaces, including but not limited to USB, Bluetooth, and serial devices. Furthermore, the Generic Sensor API allows you to read from device sensors. All capabilities discussed in this section are only available on Chromium-based browsers and on systems where the respective hardware interface or sensor is present.
+
+#### Web USB API
+The [Web USB API](https://web.dev/usb/) allows developers to access USB devices without any drivers or third-party applications. For instance, this capability is interesting for firmware updaters that developers otherwise would have to implement as separate native apps for different platforms. You need to call the `navigator.usb.requestDevice()` method to access USB devices. It takes an object which defines filters for the list of all connected USB devices. You need to specify the `vendorId` at least. The browser shows a device picker where the user can choose a matching device. From there, you can begin a device session.
+
+```js
+try {
+  const device = await navigator.usb.requestDevice({ filters: [{ vendorId: 0x8086 }] });
+  console.log(device.productName);
+  console.log(device.manufacturerName);
+} catch (err) {
+  console.log(err);
+}
+```
+
+The API is generally available on Chromium-based browsers since version 61. 182 desktop and 155 mobile sites use this API, for example, the PWA [Vysor](https://app.vysor.io/#/) that allows you to mirror the screen of an Android or iOS device—all without installing any additional software on your computer.
```

Comment:
```suggestion
The API is generally available on Chromium-based browsers since version 61. 182 desktop and 155 mobile sites use this API, for example, the PWA [Vysor](https://app.vysor.io/#/) that allows you to mirror the screen of an Android or iOS device—all without installing any additional software on your computer.
```

Reply:
GitHub tells me the "suggestion is invalid because no changes were made to the code"…